internal/pkg/describe/uri.go

```diff
@@ -220,37 +220,41 @@ type albDescriber struct {
 	svcDescriber    ecsDescriber
 	envDescriber    envDescriber
 	initLBDescriber func(string) (lbDescriber, error)
-	envDNSNameKey   string
+	envDNSTypeKey   string
```

Comment:
What do you think of renaming this to
```suggestion
	albCfnOutputName   string // The DNS name for the public or private ALB.
```